src/DDTrace/Integrations/Curl/CurlIntegration.php

```diff
@@ -148,3 +182,16 @@ public static function injectDistributedTracingHeaders($ch)
         }
     }
 }
+
+/**
+ * @param span $span
+ * @param tagName $tagName
+ * @param info $info
+ */
+ function tagFromCurlInfo(&$span, &$info, $tagName, $curlInfoOpt)
+ {
+     if (array_key_exists($curlInfoOpt, $info)) {
+        $span->setTag($tagName, $info[$curlInfoOpt]);
+        unset($info[$curlInfoOpt]);
```

Comment:
Style nit: I think it's complaining about the indent size here too (message is less clear).